src/PufferVaultV2.sol

```diff
@@ -444,6 +475,24 @@ contract PufferVaultV2 is PufferVault, IPufferVaultV2 {
         $.exitFeeBasisPoints = newExitFeeBasisPoints;
     }
 
+    // slither-disable-next-line dead-code
+    modifier markDeposit() virtual {
+        assembly {
+            tstore(_DEPOSIT_TRACKER_LOCATION, 1)
+        }
+        _;
+    }
+
+    modifier revertIfDeposited() virtual {
+        assembly {
+            if tload(_DEPOSIT_TRACKER_LOCATION) {
```

Comment:
want to confirm that tstore housekeeping is accounted for

Reply:
https://eips.ethereum.org/EIPS/eip-1153 says:
 **All values in transient storage are discarded at the end of the transaction.**